[2023-05-19T21:35:55.968+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_todays_gamesv2.call_games manual__2023-05-19T21:35:51.967778+00:00 [queued]>
[2023-05-19T21:35:55.978+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_todays_gamesv2.call_games manual__2023-05-19T21:35:51.967778+00:00 [queued]>
[2023-05-19T21:35:55.979+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-19T21:35:55.979+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-05-19T21:35:55.980+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-19T21:35:55.997+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): call_games> on 2023-05-19 21:35:51.967778+00:00
[2023-05-19T21:35:56.004+0000] {standard_task_runner.py:55} INFO - Started process 316 to run task
[2023-05-19T21:35:56.008+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_todays_gamesv2', 'call_games', 'manual__2023-05-19T21:35:51.967778+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/todays_gamesv2.py', '--cfg-path', '/tmp/tmptk5nikh3']
[2023-05-19T21:35:56.009+0000] {standard_task_runner.py:83} INFO - Job 57: Subtask call_games
[2023-05-19T21:35:56.091+0000] {task_command.py:388} INFO - Running <TaskInstance: get_todays_gamesv2.call_games manual__2023-05-19T21:35:51.967778+00:00 [running]> on host 2e718862c733
[2023-05-19T21:35:56.180+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_todays_gamesv2
AIRFLOW_CTX_TASK_ID=call_games
AIRFLOW_CTX_EXECUTION_DATE=2023-05-19T21:35:51.967778+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-19T21:35:51.967778+00:00
[2023-05-19T21:35:56.185+0000] {logging_mixin.py:137} INFO - {{ ds }}
[2023-05-19T21:35:56.446+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/todays_gamesv2.py", line 35, in _call_games
    games = schedule(start_date=date_call,end_date=date_call,team=team)
  File "/opt/airflow/dags/statsapi/__init__.py", line 94, in schedule
    r = get("schedule", params)
  File "/opt/airflow/dags/statsapi/__init__.py", line 1736, in get
    r.raise_for_status()
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://statsapi.mlb.com/api/v1/schedule?startDate=%7B%7B%20ds%20%7D%7D&endDate=%7B%7B%20ds%20%7D%7D&teamId=147&sportId=1&hydrate=decisions,probablePitcher(note),linescore,broadcasts,game(content(media(epg))),seriesStatus
[2023-05-19T21:35:56.457+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=get_todays_gamesv2, task_id=call_games, execution_date=20230519T213551, start_date=20230519T213555, end_date=20230519T213556
[2023-05-19T21:35:56.476+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 57 for task call_games (400 Client Error: Bad Request for url: https://statsapi.mlb.com/api/v1/schedule?startDate=%7B%7B%20ds%20%7D%7D&endDate=%7B%7B%20ds%20%7D%7D&teamId=147&sportId=1&hydrate=decisions,probablePitcher(note),linescore,broadcasts,game(content(media(epg))),seriesStatus; 316)
[2023-05-19T21:35:56.502+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-19T21:35:56.543+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
